core/domain/story_services.py

```diff
@@ -30,18 +30,11 @@
 datastore_services = models.Registry.import_datastore_services()
 memcache_services = models.Registry.import_memcache_services()
 
-# This takes additional 'title' parameters.
-CMD_CREATE_NEW = 'create_new'
-
-# The maximum number of iterations allowed for populating the results of a
-# search query.
-MAX_ITERATIONS = 10
-
 
 def _migrate_story_contents_to_latest_schema(versioned_story_contents):
     """Holds the responsibility of performing a step-by-step, sequential update
     of the story structure based on the schema version of the input
-    story dictionary. If any of the current story schemas
+    story dictionary. If the current story_contents schemas
```

Comment:
schemas change --> schema changes (you are talking about one schema here that evolves over time)

Reply:
Done.